frontend/components/search/CheckboxGroup.jsx

```diff
@@ -0,0 +1,60 @@
+import {
+  Checkbox,
+  FormControl,
+  FormControlLabel,
+  FormGroup,
+  FormLabel
+} from "@material-ui/core";
+import {useContext} from "react";
+import {SearchFormContext} from "./SearchForm";
+
+export default function CheckboxGroup({label, name, children}) {
+  // A component for checkbox inputs in the search form.
+  // `label` is displayed above the checkboxes.
+  // `name` is the query parameter key used in the search API request.
+  // `children` is an array of options, objects with keys `label` and `key`.
+  //    `label` is rendered next to the option.
+  //    `key` is the value of the option used in the API request.
+
+  // get current input values
+  const {state, setState, disabled} = useContext(SearchFormContext);
+
+  // If the current state is not set, default to all options.
+  // This is the default behavior expected by the API.
+  let checkedItems = state[name] || children.map(({key}) => key);
```

Comment:
Not sure why, but this is currently not working on my end

Reply:
04696702d0cb18681ab54c35b34adae0d6f64828